massa-protocol-worker/src/protocol_network.rs

```diff
@@ -535,38 +514,29 @@ impl ProtocolWorker {
         self.remove_asked_blocks_of_node(&remove_hashes)
     }
 
-    async fn on_block_info_received(
+    fn on_block_info_received(
         &mut self,
         from_node_id: NodeId,
         block_id: BlockId,
         info: BlockInfoReply,
-        op_timer: &mut Pin<&mut Sleep>,
     ) -> Result<(), ProtocolError> {
         match info {
             BlockInfoReply::Header(header) => {
                 // Verify and Send it consensus
                 self.on_block_header_received(from_node_id, block_id, header)
-                    .await
             }
             BlockInfoReply::Info(operation_list) => {
                 // Ask for missing operations ids and print a warning if there is no header for
                 // that block.
                 // Ban the node if the operation ids hash doesn't match with the hash contained in
                 // the block_header.
-                self.on_block_operation_list_received(
-                    from_node_id,
-                    block_id,
-                    operation_list,
-                    op_timer,
```

Comment:
So nice that you removed the timers in parameters !

Reply:
Thanks. Yes that is a result of not needing to pin the async stuff anymore.